docs/examples/bridge

```diff
@@ -3,5 +3,7 @@ Interface=br0
 Connection=bridge
 BindsToInterfaces=(eth0 eth1 tap0)
 IP=dhcp
+# Optionally use the MAC address of interface eth1
+MACAddressOf=eth1
```

Comment:
I suggest commenting these lines out to indicate that they are not necessary.
Also, the option needs to be added to the manpage.

Reply:
@joukewitteveen Alright, I have commented this out and added the manpage documentation